MainProject/readers/BMPReader.cs

```diff
@@ -0,0 +1,59 @@
+﻿using ImageConverter.Sdk.Interfaces;
+using ImageConverter.Sdk.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.PortableExecutable;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MainProject.Readers
+{
+    public class BMPReader : IImageReader
+    {
+        public string Title => "Kek";
+
+        public string Description => "kok";
```

Comment:
it is not a test reader. Please provide a conscious naming

Reply:
Btw, for what purpose we need Title and Description? In what case we will use it?

Reply:
In case our project will become world wide famous and other developers will be using our SDK we have to provide them with some functionality to describe their plugins